packages/parser-package-json/src/schema.json

```diff
@@ -72,8 +72,7 @@
         "patternProperties": {
           "^_": {
             "description": "Any property starting with _ is valid.",
-            "additionalProperties": true,
-            "additionalItems": true
+            "additionalProperties": true
```

Comment:
Just curious why this changed. Is `additionalItems` no longer supported in ajv 7 or was something else breaking?

Reply:
ajv fails if `additionalItems` is not in a type `array`